README.md

```diff
@@ -21,10 +21,11 @@ If you would prefer to validate the requirements manually, please verify that:
 * The package repositories are all publicly accessible.
 * The packages all contain a `Package.swift` file in the root folder.
 * The packages are written in Swift 4.0 or later.
-* The packages all contain at least one product (either library or executable).
+* The packages all contain at least one product (either library or executable), and that product must be usable in other Swift apps.
```

Comment:
I think this is the simplest way to say it. I did originally have a more wordy version, but I think this covers it.

This is what I was going to write somewhere...

> Packages must contain at least one library or executable product which is useful and usable in Swift projects that might import the package.

We can include that still, if you think this is not enough.